### PR TITLE
adding text image 99

### PR DESCRIPTION
**Pull request by JasmineDavidson94**
Created a simple text-on-image effect using HTML and CSS with the structure provided in the issue instructions. Thank you!

### CODE REVIEW

**Comment by JasmineDavidson94** [comment]
Hello please review and merge this request as soon as possible.

File: modules/text_on_image/text_on_image_99_by_Jasmine_Davidson.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>In The City</title> \\Text on Image Animation
```
Comment (krushnarout) [inline modules/text_on_image/text_on_image_99_by_Jasmine_Davidson.html]: Change the comment tag `\\Text on Image Animation` to `<!--Text on Image Animation-->`

**Review by krushnarout — CHANGES_REQUESTED** [review]

File: modules/text_on_image/text_on_image_99_by_Jasmine_Davidson.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            margin: 0;
+            padding: 5px 10px;
+            background-color: rgba(0, 0, 0, 0.5);
+        }
+    </style>
+</head>
+<body>
+
+<div class="image-container">
+    <img src="https://images01.nicepagecdn.com/page/24/32/html-template-2432144.jpg" alt="Background Image">
```
Comment (NtemKenyor) [inline modules/text_on_image/text_on_image_99_by_Jasmine_Davidson.html]: please use a nice image for your background(you can download it). Add it to your PR. You can check examples of text_on_image that have been accepted in the modules folder.

Comment (JasmineDavidson94) [reply modules/text_on_image/text_on_image_99_by_Jasmine_Davidson.html]: Thank you for the feedback! I have made the changes requested.

Comment (krushnarout) [reply modules/text_on_image/text_on_image_99_by_Jasmine_Davidson.html]: @JasmineDavidson94 Download the image and add it.

**Comment by krushnarout** [comment]
Looks good to me, Thanks for contributing

**Review by krushnarout — APPROVED** [review]

**Pull request merged**